modules/virt-defining-watchdog-device-vm.adoc

```diff
@@ -23,26 +23,27 @@ apiVersion: kubevirt.io/v1
 kind: VirtualMachine
 metadata:
   labels:
-    kubevirt.io/vm: vm2-rhel84-watchdog
+    kubevirt.io/vm: <vm-label>
   name: <vm-name>
 spec:
   runStrategy: Halted
   template:
     metadata:
       labels:
-        kubevirt.io/vm: vm2-rhel84-watchdog
+        kubevirt.io/vm: <vm-label>
     spec:
       domain:
         devices:
           watchdog:
             name: <watchdog>
-            i6300esb:
-              action: "poweroff" <1>
+            <watchdog-device-model>: <1>
+              action: "poweroff" <2>
 # ...
 ----
-<1> Specify `poweroff`, `reset`, or `shutdown`.
+<1> The watchdog device model to use. For `x86` specify `i6300esb`. For `s390x` specify `diag288`.
+<2> Specify `poweroff`, `reset`, or `shutdown`. The `shutdown` action requires that the guest virtual machine is responsive to ACPI signals. Therefore, using `shutdown` is not recommended.
```

Comment:
We try to avoid language around recommendations for the most part. This might be a case where you want it, just noting it for consideration.

Reply:
In this case it's really a recommendation see: https://docs.redhat.com/en/documentation/red_hat_enterprise_linux/6/html/virtualization_administration_guide/section-libvirt-dom-xml-watchdog